pkg/ssh/server/ssh.go

```diff
@@ -328,10 +330,13 @@ func (s *Server) getCommand(sess ssh.Session, isPty bool) *exec.Cmd {
 	}
 
 	// switch default directory
-	home, _ := command.GetHome(user)
-	_, err := os.Stat(home)
+	workdir := s.workdir
+	if workdir == "" {
+		workdir, _ = command.GetHome(user)
+	}
+	_, err := os.Stat(workdir)
 	if err == nil {
-		cmd.Dir = home
+		cmd.Dir = workdir
 	}
```

Comment:
we want to fall back to the user home directory here if the path set by `s.workdir` can't be found. Right now if `os.Stat` fails we end up in `/root` 😅